packages/migration/src/migrations/hearth/20240508142533-location-view-with-plain-ids.ts

```diff
@@ -17,22 +17,30 @@ export const up = async (db: Db, client: MongoClient) => {
     pipeline: [
       {
```

Comment:
@Zangetsu101 as this is slightly related to
https://github.com/opencrvs/opencrvs-core/issues/7180

I wonder if we could completely filter out the home addresses from this view if those are the ones without partOf properly set up?

Reply:
yea makes sense. but then the bundle won't resolve those location resources :grimacing: You know `Encounter.location.location.reference `

Reply:
Good point. Ok, let's proceed with this for now 👍

Reply:
you know what, we probably don't need to worry about that. let me try it out

Reply:
There is potentially some other bug here. :sweat_smile: The `partOf` field is not there when `{ 'type.coding.code': "OTHER" }` but it's there for the value "PRIVATE_HOME"

Reply:
Okay I get it now, it's not there for international addresses which makes sense. So let's keep it as is now